cdap-docs/reference-manual/source/cli-api.rst

```diff
@@ -141,24 +141,24 @@ These are the available commands:
    ``list spark``,"Lists all Spark Programs."
```

Comment:
Was this text changed by running the tool that generates the rst code? If so, cool. If not, you should use it, as that way it will match exactly what has been programmed for the CLI.